community/codegen/src/main/java/org/neo4j/codegen/bytecode/ByteCodeVerifier.java

```diff
@@ -349,7 +349,7 @@ protected boolean isAssignableFrom( Type target, Type value )
         protected boolean isSubTypeOf( BasicValue value, BasicValue expected )
         {
             return super.isSubTypeOf( value, expected ) || check
-                    .invokableInterface( expected.getType(), value.getType() );
+                    .invocableInterface( expected.getType(), value.getType() );
```

Comment:
What's wrong with invokable?

Reply:
It's debatable, but according to this "invocable" should be preferred: https://en.wiktionary.org/wiki/Talk:invokable